modules/local/cat.nf

```diff
@@ -1,39 +1,42 @@
 process CAT {
-    tag "${meta.assembler}-${meta.binner}-${meta.id}-${db_name}"
+    tag "${meta.assembler}-${meta.binner}-${meta.domain}-${meta.refinement}-${meta.id}-${db_name}"
 
-    conda "bioconda::cat=4.6 bioconda::diamond=2.0.6"
+    conda "bioconda::cat=5.2.3"
```

Comment:
My guess is that diamond is not needed, but was it needed to begin with?

Reply:
The version of Diamond used has to match that used to generate the CAT database - the current 'official' database is still at 2.0.6 AFAIK.

Reply:
DIAMOND is indeed in the conda package now: https://github.com/bioconda/bioconda-recipes/blob/master/recipes/cat/meta.yaml 

@maxibor updated this to fix another bug I think(?). That said I ran the pipeline with this version of CAT with no issues with this module a few weeks ago. I'm going to run again to double check now though, but otherwise we can continue IMO.

Reply:
I can confirm it runs through with v5.2.3 and with the official database so I think we are OK :+1: